libparsec/src/cli/list_devices.rs

```diff
@@ -0,0 +1,22 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use clap::Args;
+use std::path::PathBuf;
+
+use crate::{cli::*, get_default_config_dir, list_available_devices};
+
+#[derive(Args)]
+pub struct ListDevices {
+    /// Parsec config directory
+    #[arg(short, long)]
+    config_dir: Option<PathBuf>,
+}
+
+pub async fn list_devices(list_devices: ListDevices) {
+    let config_dir = list_devices.config_dir.unwrap_or(get_default_config_dir());
+    let devices = list_available_devices(&config_dir).await;
+    let n = devices.len();
+
+    println!("Found {GREEN}{n}{RESET} device(s) in {YELLOW}{config_dir:?}{RESET}:");
+    format_devices(&devices)
```

Comment:
Shouldn't we have an if/else in case of no device (`n == 0`)? 
So we can print `"No devices found in ..."`